<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RegisterProfesseur</class>
 <widget class="QDialog" name="RegisterProfesseur">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>511</width>
    <height>554</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>RegisteProfesseur</string>
  </property>
  <widget class="QGroupBox" name="RegiStudForm">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>513</width>
     <height>551</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Aldhabi</family>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">#RegiStudForm{
background :rgb(0, 170, 127)
}</string>
   </property>
   <property name="title">
    <string>Formulaire pour les Professeurs</string>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="2" column="0">
     <widget class="QLabel" name="label_2">
      <property name="font">
       <font>
        <family>Andalus</family>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Prenom</string>
      </property>
     </widget>
    </item>
    <item row="5" column="0">
     <widget class="QLineEdit" name="UsernamProfRegi"/>
    </item>
    <item row="1" column="0">
     <widget class="QLineEdit" name="NomProf"/>
    </item>
    <item row="8" column="0">
     <widget class="QPushButton" name="ProfButtomRegister">
      <property name="font">
       <font>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Registrer</string>
      </property>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QLabel" name="label">
      <property name="font">
       <font>
        <family>Andalus</family>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Nom</string>
      </property>
     </widget>
    </item>
    <item row="6" column="0">
     <widget class="QLabel" name="PasswordEtud">
      <property name="font">
       <font>
        <family>Andalus</family>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Password</string>
      </property>
     </widget>
    </item>
    <item row="4" column="0">
     <widget class="QLabel" name="label_3">
      <property name="font">
       <font>
        <family>Andalus</family>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Username</string>
      </property>
     </widget>
    </item>
    <item row="7" column="0">
     <widget class="QLineEdit" name="PasswordProfRegi">
      <property name="echoMode">
       <enum>QLineEdit::Password</enum>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QLineEdit" name="PrenomProf"/>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
